assets/js/theme/common/product-details.js

```diff
@@ -443,6 +443,38 @@ export default class ProductDetails extends ProductDetailsBase {
         this.setLiveRegionAttributes($addToCartBtn.next(), 'status', 'polite');
     }
 
+    /**
+     *
+     * Add a product to wishlist
+     *
+     */
+    addProductToWishlist() {
+        const $addToWishlistBtn = $('.productView-options .form-wishlist');
+        
+        $addToWishlistBtn.on("submit", function(event) {
+            event.preventDefault();
+
+            const url = $(event.originalEvent.submitter).attr('formaction');
+            const wishlistId = $(event.originalEvent.submitter).attr('data-wishlist-id');
+
+            $.post({url})
```

Comment:
Could we use common approach and move this call to utils.api?

Reply:
done!